build-system/server.js

```diff
@@ -504,11 +504,15 @@ function replaceUrls(mode, file) {
     file = file.replace('https://cdn.ampproject.org/a4a-v0.max.js', '/dist/amp-inabox.js');
     file = file.replace(/https:\/\/cdn.ampproject.org\/v0\//g, '/dist/v0/');
     file = file.replace('https://cdn1.ampproject.org/viewer/google/v5.js', 'https://cdn.ampproject.org/viewer/google/v5.js');
+    file = file.replace(/https:\/\/3p.ampproject.net\/(?:\d+\/)?(.+)\.html/,
```

Comment:
Sorry -- should have been more clear.  I meant, when manually loading pages for personal testing, not when running automated tests.  If I have a file that looks like,

```html
<body>
<h1>A putative ad.</h1>
<iframe src="https://3p.ampproject.net/12345678/nameframe.html"
        name='{"creative":"<html><head><meta charset=\"UTF-8\"><title>Who puts a title on an ad creative anyway?</title></head><body>This is an ad.</body></html>","context":{"foo":3.7,"bar":"kumquat"}}'></iframe>
</body>```

I can load it directly in the browser, when serving locally.